base/harmful/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
-add_library(harmful harmful.c)
-install(TARGETS harmful EXPORT global ARCHIVE DESTINATION lib)
+# Some strage segfaults if linked
+if(NOT WITH_COVERAGE)
+    add_library(harmful harmful.c)
```

Comment:
If there are segfaults because of harmful library, that means that some code is using harmful function consider to fix it instead of disabling.